packages/in17n/LICENSE.md

```diff
@@ -0,0 +1,29 @@
+# BSD 3-Clause License
```

Comment:
Why not MIT?

Reply:
This. Please make it MIT. In fact, please delete the entire file. Lerna is going to overwrite it with the LICENSE.md in root anyway.